R/flywire-skeleton.R

```diff
@@ -21,7 +22,12 @@
 #' You will need to have the \code{ncollpyde}
 #' python3 module installed. You can get this with \code{pip3 install ncollpyde}. If you get issues
 #' related to this module, best to set this to \code{FALSE}.
-#' @param radius Logical. Whether or not to return radius information for each skeleton node.
+#' @param theta numeric. Used if \code(clean==TRUE). For each twig we generate the dotproduct between the tangent
```

Comment:
@alexanderbates, this should be `\code{clean=TRUE}`. Can you fix?

Reply:
If you don't feel like running R CMD check before pushing (which you should), you should at least run `devtools::check_man()` to make sure that the docs aren't broken!